contracts/interfaces/ITransferManager.sol

```diff
@@ -13,4 +13,8 @@ interface ITransferManager {
         TransferManagerEnums.Result
```

Comment:
I wonder whether we should having something like:
`executeTransfer` (non-view) and `verifyTransfer` (view) rather than overloading `verifyTransfer`